src/components/ContributorThumbnail.js

```diff
@@ -152,14 +152,13 @@ const Thumbnail = ({
         {organization.depth === 4 &&
         checkboxValue &&
         organization.cti_contributor ? (
-            <img
-              className={classes.contributorIcon}
-              src='/images/Child_contributed.svg'
-              alt='contributor-icon'
-            />
-          ) : (
-            null
-          )}
+          <img
+            alt='contributor-icon'
+            data-cy='contributor-icon'
+            className={classes.contributorIcon}
+            src='/images/Child_contributed.svg'
+          />
+        ) : null}
```

Comment:
Prettier and eslint have conflicting rules on indentation here.  Which one takes precedence?

Reply:
We've been having conversation about this tonight. No great answers. Going to merge even with indent lint warnings for now.